server.js

```diff
@@ -16,9 +17,87 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minLength: 5,
+    maxLength: 140,
+    trim: true,
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: () => new Date(),
+  },
+});
+
+const Thought = mongoose.model('Thought', ThoughtSchema);
+
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+  res.send({
+    'Hello!': "This is Sofie's Happy Thoughts API.",
+    'Here you can watch the API live':
+      'https://graceful-lily-1fde84.netlify.app/',
+    'And see all endpoints here': '/endpoints',
+  });
+});
+
+app.get('/endpoints', (req, res) => {
+  res.send(allEndpoints(app));
+});
```

Comment:
nice route explaining what this is and also that you added the allEndpoints, makes my job easier when reviewing :)